### PR TITLE
feat: Add HTML, CSS and JS for mobile view

### LINKED ISSUE #74: Create navigation for queried mobile view

### LINKED ISSUE #115: Print Media Query Font-Size is too large.
In @media print CSS rule, font-size of .recipe-popup needs to be changed to 1rem and font-size of .print-container needs to be changed to 1.1rem.

### PR DESCRIPTION
**Pull request by jsahim**
**Files changed:** index.html, scripts.js, styles.css
**Change summary:** I added CSS styling for a mobile view using a media query. I also added a toggleMobileMenu function to handle the toggling of the new close and menu icons added in HTML. Also adjusted font sizing on print query

close #74 close #115

### CODE REVIEW

**Review by LordSchwifty — APPROVED** [review]
Love the hamburger menu and mobile view implementation overall. Good work!

File: src/scripts.js [hunk 1]
```diff
@@ -206,4 +215,16 @@ function goHome() {
         document.getElementById('filterPlaceholder').innerText = 'Filter Recipes';
         createRecipeCards(currentView);
     }
+}
+
+function toggleMobileMenu(e){
```
Comment (jfogiato) [inline src/scripts.js:220]: Great idea hiding certain elements and utilizing existing functions!

File: src/styles.css [hunk 2]
```diff
@@ -279,4 +283,121 @@ li {
 
 @page {
   margin: 2cm;
-}
\ No newline at end of file
+}
+
+@media only screen and (max-device-width: 600px){
```
Comment (jfogiato) [inline src/styles.css:288]: STELLAR work on this, Jeff! Thanks for taking it on!

**Review by jfogiato — APPROVED** [review]

File: src/styles.css [hunk 3]
```diff
@@ -263,7 +267,7 @@ li {
   }
 
   .print-container {
-    font-size: 1.5rem;
+    font-size: 1.1rem;
```
Comment (qrispi) [inline src/styles.css:270]: Thanks for fixing this!

File: src/scripts.js [hunk 4]
```diff
@@ -6,19 +6,25 @@ import './images/user.png';
 import './images/wc-logo.png';
 import './images/home-button.png';
 import './images/close-icon.png';
+import './images/hamburger-menu.png';
```
Comment (qrispi) [inline src/scripts.js:9]: Love the hamburger icon!!

**Review by qrispi — APPROVED** [review]
Impressive work on this Jeff. Can't wait to see it work once we get this activated on github pages!

**Pull request merged**